app/models/profile.rb

```diff
@@ -48,7 +48,8 @@ class Profile < ApplicationRecord
     :url => ':url_root/:account_id/:class/:attachment/:style/:basename.:extension'.freeze,
     :s3_permissions => 'public-read'.freeze,
     :default_url => '/assets/3scale-logo.png'.freeze
-  do_not_validate_attachment_file_type :logo
+  # do_not_validate_attachment_file_type :logo
+  validates_attachment_content_type :logo, content_type: /^image\/(png|gif|jpeg)/
```

Comment:
@Martouta more and more logo's are vector format these days so we should consider accepting svg. Not sure on the security implications.

Reply:
@thomasmaas Ok, I will check 👍